ui/page/governance/proposals_page.go

```diff
@@ -264,6 +271,57 @@ func (pg *ProposalsPage) Layout(gtx C) D {
 	)
 }
 
+func (pg *ProposalsPage) layoutMobile(gtx layout.Context) layout.Dimensions {
+	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+		layout.Rigid(func(gtx layout.Context) layout.Dimensions {
+			return layout.Inset{Right: values.MarginPadding10}.Layout(gtx, pg.layoutSectionHeader)
+		}),
+		layout.Flexed(1, func(gtx C) D {
+			return layout.Inset{Top: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
+				return layout.Stack{}.Layout(gtx,
+					layout.Expanded(func(gtx C) D {
+						return layout.Inset{Top: values.MarginPadding60}.Layout(gtx, pg.layoutContent)
+					}),
+					//TODO: temp removal till after V1
+					// layout.Expanded(func(gtx C) D {
+					// 	gtx.Constraints.Max.X = gtx.Dp(values.MarginPadding150)
+					// 	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+
+					// 	card := pg.Theme.Card()
+					// 	card.Radius = decredmaterial.Radius(8)
+					// 	return card.Layout(gtx, func(gtx C) D {
+					// 		return layout.Inset{
+					// 			Left:   values.MarginPadding10,
+					// 			Right:  values.MarginPadding10,
+					// 			Top:    values.MarginPadding2,
+					// 			Bottom: values.MarginPadding2,
+					// 		}.Layout(gtx, pg.searchEditor.Layout)
+					// 	})
+					// }),
+					layout.Expanded(func(gtx C) D {
+						gtx.Constraints.Min.X = gtx.Constraints.Max.X
+						return layout.E.Layout(gtx, func(gtx C) D {
+							card := pg.Theme.Card()
+							card.Radius = decredmaterial.Radius(8)
+							return layout.Inset{Right: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
+								return card.Layout(gtx, func(gtx C) D {
+									return layout.UniformInset(values.MarginPadding8).Layout(gtx, pg.layoutSyncSection)
```

Comment:
These mobile layouts are just a bunch or repeated codes. Lets have them optimized as mentioned above

Reply:
For now they might look like they are, but they are not. currently we are trying to share the same layouts for desktop and mobile, and making some optimizations to the code in order to make it look somewhat presentable on mobile. However, we are going to be implementing totally different UIs for mobile further down the line when proper mockups become available. The mobile and desktop views need to be clearly distinguishable.

This PR, the ones before it, and the ones coming after it, are aimed at setting the bedrock foundation for the godcr mobile builds

Having said that the current layout separation is in line with the scope of this PR. We should leave as is.

Reply:
Then i believe it wont be out of order to properly comments these codes for easier reviews.